bin/meshroom_photogrammetry

```diff
@@ -43,8 +46,34 @@ if not args.input and not args.inputImages:
     print('Nothing to compute. You need to set --input or --inputImages.')
     exit(1)
 
-graph = multiview.photogrammetry(inputFolder=args.input, inputImages=args.inputImages, output=args.output)
-graph.findNode('DepthMap_1').downscale.value = args.scale
+if  args.input and os.path.isfile(args.input):
+    # with open(args.input) as jsonFile:
+    with io.open(args.input, 'r', encoding='utf-8', errors='ignore') as jsonFile:
```

Comment:
@yann-lty this is partially taken from here
https://github.com/alicevision/meshroom/blob/develop/meshroom/nodes/aliceVision/CameraInit.py#L139
I can refactor this in a proper function but i don't know where to put it.
Maybe in camera init and then import it here?

Reply:
Yes it would be great to factorize this "conversion" code (from and to sfmData JSON format) as dedicated functions in CameraInit.

Reply:
Shall i make the function inside the cameraInit node and import it from there or is there a more generic module where it can go?